recipes/python/patches/ctypes-find-library.patch

```diff
@@ -0,0 +1,21 @@
+--- Python-2.7.2.orig/Lib/ctypes/util.py	2011-06-11 16:46:24.000000000 +0100
++++ Python-2.7.2/Lib/ctypes/util.py	2015-03-26 13:45:33.322172603 +0000
+@@ -83,7 +83,17 @@
+             except ValueError:
+                 continue
+         return None
+-
++elif os.name == "posix" and sys.platform == "linux3":
```

Comment:
okay i was not that keen on dropping that much code, as there are a lot of conditions but perhaps its okay just will make the patch a lot bigger.
Will see if i can get this done at the weekend